docs/usage/macros.rst

```diff
@@ -4,44 +4,67 @@ CMake Macros
 spl_add_component
 ^^^^^^^^^^^^^^^^^
 
-Add a component to a variant.
+Add a component to the build and link it with the main project.
 
 .. code-block:: cmake
 
-  spl_add_component(<relative path from project root to the component's directory>)
+ spl_add_component(<component_path>)
 
-This macro is intended to be used in the variant's parts.cmake file to add a component to the variant.
+This macro is intended for CMake-based projects to add a component to the build, compile it, and link it with the main project. The component is typically a separate subdirectory containing its own CMakeLists.txt.
```

Comment:
Same here: there is no main project and building and linking is not the main purpose of this macro. I would like to have the old description back.